### DISCUSSION TITLE
Charging starts after a while even though the limit is lower - MacBook Pro 2015 13''

### DISCUSSION THREAD

**Discussion by rolandovoces** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#741] [Q&A]
After the update to the newest version the charging starts after a while after shutdown even though the batt level on 80 and the limit is on 70. Where I can get the older version?

**Comment by MatthiasKerbl** [comment]
Hi @rolandovoces,

Older versions are available here: https://github.com/davidwernhart/AlDente/releases

Cheers,
Matthias
